number-of-covild-19-vaccinations/number-of-covild-19-vaccinations.js

```diff
@@ -279,19 +287,32 @@ if (config.widgetFamily === 'large') {
     imageStack1.addSpacer()
     column.addSpacer(2)
     
-    const total1 = parseInt((result.states[selectedState].total / 1000).toFixed(0)).toLocaleString('de')
+    let total1 = (result.states[selectedState].total / 1000).toFixed(0)
+    let total1unit = " Tsd."
+    // if total is a million or more, format as millions and not thousands
+    if ( result.states[selectedState].total > 999999 ){
+      total1 =  (result.states[selectedState].total / 1000000).toLocaleString(Device.language(), {maximumFractionDigits: 1})
+      total1unit = " Mio."
+    }
```

Comment:
Ich hatte bewusst beide Werte in Tausend angezeigt, damit man eine bessere Relation hat.
Wenn links in Tausend und rechts in Millionen die Zahlen steht, muss man kurz nachrechnen.
Wenn beide Tausend sind sieht man schneller das Verhältnis.

Du kannst aber gerne oben im Script einen Boolean einfügen über den man die Mio. Darstellung aktivieren kann und denn Boolean mit im If-Statement aufnehmen.

Was hältst du von dem Vorschlag?

Reply:
@ThisIsBenny,
das mit den doppelten scripting.app variablen hatte ich nicht bemerkt. Ist gefixed.

Mit den Einheiten hat wohl jeder eigene Vorlieben. Mit ist z.B. auch ohne nachrechnen sofort klar, welchen Anteil 18 Tsd. an 8 Mio. haben. Hingegen muss ich schon nachdenken, wenn ich eine Zahl wie "10.104t" sehe… Wenn beide Werte > 999999 sind, wird auch bei beiden die Einheit "Mio." angezeigt.

Die alternativen Einheiten sind ganz nach deinem Wunsch aber optional. Als default wieder Dein "t". 
Um meine Alternative zu nutzen, einfach als Widget "Parameter" z.B. "Berlin, 1" oder auch nur ",1" mitgeben…
Hoffe das passt jetzt und dass ich (wg. der Uhrzeit) nicht noch nen Fehler eingebaut habe ;-)